src/main/resources/chains/testnet-internal-gac-chain.conf

```diff
@@ -149,3 +149,5 @@
     # empty means that everybody can mine
     allowed-miners = []
 }
+
+include "chains/testnet-allowed-miners"
```

Comment:
Should we include the same list of the nomad deployer or will different keys be generated there?

Reply:
it's a draft. only reason it's open is for hydra to pick it up.

Reply:
(the previous attempt simply didn't work; hocon documentation isn't super clear)